cli/code_scanner.py

```diff
@@ -610,10 +645,12 @@ def get_commit_range_modified_documents(path: str, from_commit_rev: str, to_comm
 
         file_content = sca_code_scanner.get_file_content_from_commit(repo, from_commit_rev, diff_file_path)
         if file_content is not None:
+            print_click_secho(f"Read file content from commit: {diff_file_path}")
```

Comment:
you can just use the logger instead

Reply:
It's should be printed always not by logger by click.secho